app/controllers/vm_common.rb

```diff
@@ -298,15 +298,14 @@ def snap_pressed
     @active = @snap_selected.current? if @snap_selected
     @center_toolbar = 'x_vm_snapshot'
     @explorer = true
-    render :update do |page|
-      page << javascript_prologue
-      page << javascript_reload_toolbars
-
-      page.replace("flash_msg_div", :partial => "layouts/flash_msg")
-      page << "miqScrollTop();" if @flash_array.present?
-      page.replace("desc_content", :partial => "/vm_common/snapshots_desc",
-                                   :locals  => {:selected => params[:id]})
+    formatted_time = format_timezone(@snap_selected[:create_time].to_time, Time.zone, "view")
+    if @snap_selected[:total_size].to_i == 0
+      formatted_size = ''
+    else
+      num_bytes = number_to_human_size(@snap_selected[:total_size], :precision => 2)
+      formatted_size = _("%{number} bytes") % {:number => number_with_delimiter(num_bytes, :delimiter => ",", :separator => ".")}
```

Comment:
This still isn't right...

`@snap_selected[:total_size]` is the bytes.  `number_to_human_size` takes that and converts into human readable form. `number_with_delimiter` should takes the bytes, and it will output it in fully qualified comma+decimal form.

The final output on the screen should be something like `1.5 GB (1,610,612,736.0 bytes)`